share/locale/languages.json

```diff
@@ -0,0 +1,72 @@
+{
+    "af": "Afrikaans",
+    "ar": "العربية",
+    "ar_DZ": "\u200e(العربية (الجزائر",
+    "ar_EG": "\u200e(العربية (مصر",
+    "ar_SD": "\u200e(العربية (السودان" ,
+    "ast": "Asturianu",
+    "be": "Беларуская мова",
+    "bg": "български",
+    "br": "Brezhoneg",
+    "ca": "Català",
+    "ca@valencia": "Valencià",
+    "cs": "Čeština",
+    "cy": "Cymraeg",
+    "da": "Dansk",
+    "de": "Deutsch",
+    "el": "ελληνικά",
+    "en_GB": "English (GB)",
+    "en_US": "English (US)",
+    "eo": "Esperanto",
```

Comment:
fwiw, these languages don't appear to match up with the languages we have for AU3. Might not be an issue but I thought it'd be worth noting.